mlkem/polyvec.c

```diff
@@ -85,9 +85,9 @@ void polyvec_basemul_acc_montgomery_cached(poly *r, const polyvec *a,
   unsigned i;
   poly t;
 
-  POLYVEC_BOUND(a, 4096);
-  POLYVEC_BOUND(b, NTT_BOUND);
-  POLYVEC_BOUND(b_cache, MLKEM_Q);
```

Comment:
This should have never worked .... thanks for fixing that.

Reply:
It does work, but it is a mismatch with the precondition which requires nothing about b and b_cache